independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/Methods.java

```diff
@@ -186,6 +186,13 @@ public MethodVisitor visitMethod(int access, String name, String descriptor, Str
                         classLevelBindings, bytecodeTransformerConsumer, transformUnproxyableClasses));
```

Comment:
OK, my bad, I didn't know interceptors also applied to superclass methods, otherwise I'd have suggested to handle interface default methods too. Great move.